src/libraries/externals.csproj

```diff
@@ -9,7 +9,7 @@
     <BinPlaceNative>true</BinPlaceNative>
     <BinPlaceRuntime>false</BinPlaceRuntime>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <UseLiveBuiltDotNetHost Condition="'$(TargetArchitecture)' == 's390x' or '$(TargetArchitecture)' == 'armv6'">true</UseLiveBuiltDotNetHost>
+    <UseLiveBuiltDotNetHost Condition="'$(DotNetBuildFromSource)' == 'true' or '$(TargetArchitecture)' == 's390x' or '$(TargetArchitecture)' == 'armv6'">true</UseLiveBuiltDotNetHost>
```

Comment:
Yup, that was my intent. I think long term we should force building the host in all cases. It's small and very cheap compared to building the CLR. The additional build complexity of pulling from an external package is not worth it, IMO.

Reply:
> we should force building the host in all cases.

I agree. Especially when singlefilehost builds as part of clr subset these days, it makes more sense (it will also help community supported platforms). Ideally we should dogfood live corehost components in all such cases (https://github.com/dotnet/runtime/pull/58952 proves that it is possible). 🙂

Reply:
> Yup, that was my intent. I think long term we should force building the host in all cases. It's small and very cheap compared to building the CLR. The additional build complexity of pulling from an external package is not worth it, IMO.

Absolutely agree. It just needs someone doing it :)

Reply:
It's on my personal backlog, I just keep getting distracted :)